src/sentry/static/sentry/app/views/settings/projectDataPrivacy/projectDataPrivacyActions.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import Button from 'app/components/button';
+import {t} from 'app/locale';
+
+type Props = {
+  onCancel: () => void;
+  onSave: () => void;
+};
+
+const ProjectDataPrivacyActions = ({onCancel, onSave}: Props) => (
+  <Wrapper>
+    <StyledButton onClick={onCancel}>{t('Cancel')}</StyledButton>
+    <StyledButton priority="primary" onClick={onSave}>
+      {t('Save Rules')}
+    </StyledButton>
+  </Wrapper>
+);
+
+export default ProjectDataPrivacyActions;
+
+const Wrapper = styled('div')`
+  display: flex;
+  justify-content: flex-end;
+  > * {
+    margin-right: 14px;
```

Comment:
Can we use one of the sizes in `space` util?

Reply:
it won't be exactly the same as in the mockups, but I will update it ;-)

Reply:
I'm pretty sure @doralchan would prefer you use space() over sticking to the design mocks precisely.